projects/optic/src/commands/login/login.ts

```diff
@@ -18,18 +18,24 @@ export const registerLogin = (cli: Command, config: OpticCliConfig) => {
     .action(errorHandler(getLoginAction(config), { command: 'login' }));
 };
 
+export async function identifyLoginFromToken(token: string) {
+  const newClient = createOpticClient(token);
+  const result = await newClient.verifyToken();
+
+  if (result.user) {
+    alias(result.user.userId);
```

Comment:
I'm a bit confused by this. Reading the doc it looks like we would want to do:
```
identify(userId)
people.set({ '$email': email })
```

maybe I'm missing sthg

https://docs.mixpanel.com/docs/tracking/reference/javascript-full-api-reference#mixpanelpeopleset